liquibase-core/src/main/java/liquibase/integration/commandline/Main.java

```diff
@@ -1767,13 +1768,27 @@ protected void doMigration() throws Exception {
                 Scope.getCurrentScope().getUI().sendMessage(checkSum.toString());
                 return;
             } else if (COMMANDS.DB_DOC.equalsIgnoreCase(command)) {
+                // TODO - is it ok for us to re-use schemas which is optional parameter for diff
```

Comment:
Wanted to verify if it's ok to re-use existing parameters from other commands.

Reply:
I'll have to take a look further if that is done elsewhere.  I don't think it is.

Reply:
Understandable, just figured if the code was already there to handle --schemas and the usage was the same that it'd be OK. If not, that's OK too, just more additions.

Reply:
@molivasdat - Any further thoughts on this?

Reply:
@molivasdat - sorry to pester here, just would like to understand the next steps as to how to resolve this open question so we could proceed.

Reply:
@molivasdat - it's still not clear to me what the next steps are here. Any feedback you could provide would be appreciated.

Reply:
Sorry for the slow responses. 

The --schemas usage is great